mods/ts/rules/husks.yaml

```diff
@@ -0,0 +1,73 @@
+DSHP.Husk:
+	Inherits: ^AircraftHusk
+	Tooltip:
+		Name: Dropship
+	Aircraft:
+		ROT: 5
+		Speed: 168
+	RenderVoxels:
+		Image: dshp
+
+ORCA.Husk:
+	Inherits: ^AircraftHusk
+	Tooltip:
+		Name: Orca Fighter
+	Aircraft:
+		ROT: 5
+		Speed: 186
+	RenderVoxels:
+		Image: orca
+
+ORCAB.Husk:
+	Inherits: ^AircraftHusk
+	Tooltip:
+		Name: Orca Bomber
+	Aircraft:
+		ROT: 5
+		Speed: 96
+	RenderVoxels:
+		Image: orcab
+
+ORCATRAN.Husk:
+	Inherits: ^AircraftHusk
+	Tooltip:
+		Name: Orca Transport
+	Aircraft:
+		ROT: 5
+		Speed: 84
+	RenderVoxels:
+		Image: orcatran
+
+TRNSPORT.Husk:
+	Inherits: ^AircraftHusk
+	Tooltip:
+		Name: Carryall
+	Aircraft:
+		ROT: 5
+		Speed: 149
+	RenderVoxels:
+		Image: trnsport
+
+SCRIN.Husk:
+	Inherits: ^AircraftHusk
+	Tooltip:
+		Name: Banshee Fighter
+	Aircraft:
+		ROT: 5
+		Speed: 168
+	RenderVoxels:
+		Image: scrin
+
+APACHE.Husk:
+	Inherits: ^AircraftHusk
+	Tooltip:
+		Name: Harpy
+	Aircraft:
+		ROT: 5
+		Speed: 130
+	WithSpriteRotorOverlay:
+		Offset: 85,0,384
+	RenderSprites:
+		Image: apache
```

Comment:
Can you define it for the others too? This will make life easier when we add smoke and other effects.